rosdep/base.yaml

```diff
@@ -4994,6 +4994,18 @@ r-base-dev:
   fedora: [R-devel]
   gentoo: [dev-lang/R]
   ubuntu: [r-base-dev]
+range-v3:
+  debian:
+    '*': [librange-v3-dev]
+    jessie: null
```

Comment:
We don't need to add jessie here, as it is out-of-support.

Reply:
I added it because it's one of the _recommended_ supported platforms for Kinetic and I guessed it would be nice to include. It's gone now.

Reply:
Yeah, we're a bit inconsistent there.  We support these other Debian/Ubuntu versions when we first release the ROS distributions.  Once they go out of support, we stop making changes to rosdep keys for them, but we don't remove them from the REP.  Anyway, looks good to me now, thanks for iterating.

Reply:
Got it. Thanks for the explanation :+1: